include/onnxruntime/core/optimizer/graph_transformer.h

```diff
@@ -9,15 +9,33 @@
 
 namespace onnxruntime {
 
+/**
+* Default_Global refers to all the transformations which are required for 
+  corrcetness and are execution provider independent
+* Default_ProvderSpecific refers to transformations which are also required 
+  for correctness but are execution provider specific
+* Optional_L1 refers to all basic optmizations. These are run both before and 
+  after graph partitioning
+* Optional_L2 refers to all fusion optmizations. These are run after graph 
```

Comment:
>fusion [](start = 28, length = 6)

Is it required that they be fusion optimizations?

Reply:
No it is not required. Let me refactor this and add fusion as an example.